### PR TITLE
EDSC-3776: Create SNWG Portal

### PR DESCRIPTION
**Pull request by DanielJDufour**
# Overview

### What is the feature?

Created prototype of portal for Satellite Needs Working Group.

### What is the Solution?

- created snwg folder inside of portals
- inside of snwg folder, created config.json

### What areas of the application does this impact?

- portals

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. go to main page
2. click browse portals
3. click SNWG portal card

### Attachments

Screenshot 2023-05-24 at 4 59 11 PM

Screenshot 2023-05-24 at 5 01 07 PM

# Checklist

- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: portals/snwg/config.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "moreInfoUrl": "https://www.earthdata.nasa.gov/esds/impact/snwg",
+  "pageTitle": "SNWG",
+  "parentConfig": "edsc",
+  "portalBrowser": true,
+  "query": {
+    "conceptId": [
```
Comment (eudoroolivares2016) [inline portals/snwg/config.json:7]: Double checked all of these against the google doc on the ticket looks like they're all in there

Comment (DanielJDufour) [reply portals/snwg/config.json:7]: thanks for the quick look, Ed!

**Review by eudoroolivares2016 — APPROVED** [review]

**Review by trevorlang — APPROVED** [review]

**Review by macrouch — CHANGES_REQUESTED** [review]
This is missing something from the portal creation guide but I'm not on my laptop now to find it

**Comment by macrouch** [comment]
https://wiki.earthdata.nasa.gov/display/EDSC/Portal+Creation+Guide#PortalCreationGuide-AddingPortaltotheindex.jslist

I also don't think we want to add all those conceptIds

File: portals/snwg/config.json [hunk 2]
```diff
@@ -4,65 +4,8 @@
...
-      "C2548345108-NSIDC_ECS",
-      "C2551528419-NSIDC_ECS",
-      "C2551529078-NSIDC_ECS",
-      "C2204095765-NSIDC_ECS",
-      "C2517904291-LPCLOUD",
-      "C2429062328-POCLOUD"
-    ],
-    "hasGranulesOrCwic": null
+    "hasGranulesOrCwic": null,
+    "tagKey": "org.snwg"
```
Comment (eudoroolivares2016) [inline portals/snwg/config.json:8]: 👍

**Pull request merged**